CZI_mpl.tex

```diff
@@ -150,56 +154,63 @@ \subsubsection{API generalization}
 around the concept of a global ``current Axes''.  While convenient for
 quick interactive usage, it frequently produces surprising and
 undesired coupling between in the code when used in libraries.  On the
+% Something is missing in the line above, but I don't know what it is...
 other hand the \texttt{OO} interface is more explicit and flexible but
-more verbose.  However, the main name space for plotting methods is on
-the \texttt{Axes} object which leads to three issues with the API.
+more verbose.  The main name space for plotting methods is
+the \texttt{Axes} class, which leads to three issues with the API.
 First, third party domain-specific packages can never feel ``First
-Class'' as they will not be implemented as \texttt{Axes} methods like
+Class'', as their plotting functions will not be implemented as \texttt{Axes} methods like
 the ``built in'' plots.  Second, some visualizations require putting
-\texttt{Artist}s on multiple \texttt{Axes} which can not be naturally
-expressed as an \texttt{Axes} method.  Lastly, there are over 250
-methods on the \texttt{Axes}, all of the plotting methods and some
-additional \texttt{Axes} specific methods, which makes it extremely
-hard to discover if the method you need exists via tab-completion.
-
-To address these issues we will move to top-level functions that take
-in data, style, and \texttt{Axes}s as the primary API.  During this
+\texttt{Artist}s on multiple \texttt{Axes}; this doesn't fit the model
+in which Artists are made via \texttt{Axes} methods.  Lastly, there are over 250
+methods on the \texttt{Axes}---all of the plotting methods and some
+additional \texttt{Axes} specific methods---which makes it extremely
+hard to use tab-completion to search for a method.
+
+To address these issues we will move to a primary API in which top-level
+functions take
+in data, style, and \texttt{Axes}s.  During this
 refactoring we will use consistent naming and call conventions, as
 discussed in \ref{sec:api_hom}, that can be used by downstream
 libraries.  These functions will return the rich composites discussed
 in section \ref{sec:artists} and consume the data objects discussed in
 section \ref{sec:dm}.  From the large pool of plotting functions we
 will curate domain-specific name spaces, which can be augmented by
-down stream libraries to enable users to discover the functionality
+downstream libraries to enable users to discover the functionality
 they need.
 
 This is a huge change in the API that will require time to carefully
 consider the consequences.
+% Does this mean it the consequences might be so bad that it won't be
```

Comment:
This is trying to make the point that this is out of scope for what can be done on volunteer time.